### PR TITLE
Update Framework Options

### PR DESCRIPTION
**Pull request by ghost**
See github issue: https://github.com/octobercms/october/pull/4285

### CODE REVIEW

File: markup-filter-theme.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The following aliases are supported:
 
 Alias | Description
 ------------- | -------------
-`@jquery` | Reference to the jQuery library (v3.4.0) used in the back-end. (JavaScript)
+`@jquery` | Reference to the jQuery library (latest available version) used in the back-end. (JavaScript)
```
Comment (LukeTowers) [inline markup-filter-theme.md]: keep the version number please, it's a good quick reference for developers to know what version we use.

File: ajax-introduction.md [hunk 2]
```diff
@@ -21,9 +22,22 @@ The AJAX framework is optional in your [CMS theme](../cms/themes), to use the li
...
+
+    {% framework %}
     {% framework extras %}
+    {% framework jquery %}
+
+Field | Description
+------------- | -------------
+**framework** | loads the file `framework.js`.
+**framework extras** | the `{% framework %}` tag supports the optional **extras** parameter. If this parameter is specified, the tag adds StyleSheet and JavaScript files for [extra features](../ajax/extras), including form validation and loading indicators. It will load the following asset files: `framework.js`, `framework.extras.js` and `framework.extras.css`.
+**framework jquery** | this will load the latest version of jQuery and jQuery Migrate (adding backwards compatibility). It also includes a `Subresource Integrity (SRI)` attribute, this is a security feature that allows your browser to distinguish if the files being retrieved have been maliciously altered.
```
Comment (LukeTowers) [inline ajax-introduction.md]: This should just be jquery I believe

**Review by LukeTowers — CHANGES_REQUESTED** [review]

File: ajax-introduction.md (hunk 2 above)
Comment (LukeTowers) [reply ajax-introduction.md]: huh, just checked and it doesn't look like there is a jQuery tag. You can just get rid of that entirely including the jquery stuff in the framework twig node

**Comment by daftspunk** [comment]
Might revisit this when there are more options

**Pull request closed without merging**